proto/cmp/services/partner/v2/partner_configuration.proto

```diff
@@ -35,6 +35,8 @@ message GetPartnerConfigurationResponse {
 // ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/partner/v2/partner_configuration.proto.dot.xs.svg)
 //
 // [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/partner/v2/partner_configuration.proto.dot.svg)
+//
+/// @custom:cmp-service type:core routing:local on-chain:true
```

Comment:
not sure if this is type:core or type:service

Reply:
We don't have `type:service`. Do you mean `type:system`?

Reply:
I meant type:product

Reply:
Switched to `type:product`